cli/src/build-settings.ts

```diff
@@ -28,8 +28,13 @@ export default class BuildSettings {
   }
 
   static current(toolchain: rust.Toolchain) {
-    let rustc = rust.spawnSync("rustc", ["--version"], toolchain)
-      .stdout
+    const stdout = rust.spawnSync("rustc", ["--version"], toolchain).stdout;
+
+    if (!stdout) {
+      throw new Error('Rust is not installed or rustc is not in your path.');
```

Comment:
Maybe better to store and print original message from `stderr`? (or is it useless?)

Reply:
Good idea, let me check.

Reply:
The original message just contains "spawnSync rustc ENOENT" so that isn't too helpful. Nevertheless, if another error beside `ENOENT` appears, it will also be thrown.